.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+sudo: required
+dist: trusty
+python:
+ - "2.7"
+
+# https://stackoverflow.com/a/35029430/6605826
+
+virtualenv:
+  system_site_packages: true
+
+install:
+ - sudo apt-get update
```

Comment:
The `apt-get` stuff wouldn't be needed if the installation could be accomplished purely by pip.